### PR TITLE
GPXSee: new recipe

### PR DESCRIPTION
**Pull request by sikmir**
virtualbox_haiku_01_10_2018_23_07_33

### CODE REVIEW

File: sci-geosciences/gpxsee/gpxsee-6.0.recipe [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	gpxsee$secondaryArchSuffix = $portVersion
+	app:GPXSee = $portVersion
+	"
+REQUIRES="
+	haiku${secondaryArchSuffix}
```
Comment (diversys) [inline sci-geosciences/gpxsee/gpxsee-6.0.recipe]: You can drop curly brackets here, it is mostly needed for haiku_devel package.

File: sci-geosciences/gpxsee/gpxsee-6.0.recipe [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+	devel:libGL$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5PrintSupport$secondaryArchSuffix
+	devel:libQt5Sql$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:g++${secondaryArchSuffix}
```
Comment (diversys) [inline sci-geosciences/gpxsee/gpxsee-6.0.recipe]: Drop curly brackets too

File: sci-geosciences/gpxsee/gpxsee-6.0.recipe [hunk 3]
```diff
@@ -0,0 +1,78 @@
+SUMMARY="GPS log file viewer and analyzer"
+DESCRIPTION="GPXSee is a Qt-based GPS log file viewer and analyzer that supports \
+GPX, TCX, KML, FIT, IGC, NMEA, SLF, LOC and OziExplorer files."
+HOMEPAGE="http://www.gpxsee.org/"
```
Comment (fbrosson) [inline sci-geosciences/gpxsee/gpxsee-6.0.recipe]: Suggestion:
```diff
-HOMEPAGE="http://www.gpxsee.org/"
+HOMEPAGE="http://www.gpxsee.org/"
```

File: sci-geosciences/gpxsee/gpxsee-6.0.recipe (hunk 1 above)
Comment (sikmir) [reply sci-geosciences/gpxsee/gpxsee-6.0.recipe]: Updated.

File: sci-geosciences/gpxsee/gpxsee-6.0.recipe (hunk 2 above)
Comment (sikmir) [reply sci-geosciences/gpxsee/gpxsee-6.0.recipe]: Updated.

File: sci-geosciences/gpxsee/gpxsee-6.0.recipe (hunk 3 above)
Comment (sikmir) [reply sci-geosciences/gpxsee/gpxsee-6.0.recipe]: Mmm, what's the difference?

**Pull request merged**